spec/generator.json

```diff
@@ -31,10 +31,19 @@
             ]
         },
 
+        "source_file" {
+            "description": "Represents a source file",
+            "fields": [
+                { "name": "filename", "type": "string", "description": "The recommended name for the file, including the path to the file, where appropriate.", "example": "com/myorg/myapi/Foo.java" },
+                { "name": "source", "type": "string", "description": "The actual source code." }
```

Comment:
Discussed this with both of you in person, but wanted to document it here. 

I think apidoc should be responsible for formatting the output as the user requests it, and putting it in a .zip file is just one of the various options. The generators should only need to return normal JSON to apidoc.